hack/quickstart/quickstart-gce.md

```diff
@@ -8,6 +8,28 @@ This can be changed to identify separate clusters.
 export CLUSTER_PREFIX=quickstart
 ```
 
+### Add your SSH public key to project under user core
+
+Quickstart scripts assume you can ssh into the `core` user. Either follow the commands below or go to the web console to add a key for the user `core`.
+
+**WARNING** The following process will clobber existing keys in your project's ssh metadata.
```

Comment:
Yeah I discovered the side effect while testing. I don't have strong feelings really either way on which PR to go forth with, but it seemed simpler in my mind to just require the core user upfront when people are using gce-quickstart.